starport/services/network/address.go

```diff
@@ -0,0 +1,14 @@
+package network
+
+import (
+	"github.com/cosmos/cosmos-sdk/types/bech32"
+)
+
+// SetSPNPrefix change the address prefix to the spn prefix
+func SetSPNPrefix(address string) (string, error) {
+	_, pubKey, err := bech32.DecodeAndConvert(address)
```

Comment:
Can we move the logic of this func to cosmosaccount.ConvertPrefix(address, newPrefix)?